packages/base/src/3dview/mainview.tsx

```diff
@@ -1411,6 +1441,9 @@ export class MainView extends React.Component<IProps, IStates> {
   private _controls: OrbitControls; // Mouse controls
   private _transformControls: TransformControls; // Mesh position/rotation controls
   private _pointer3D: IPointer | null = null;
+  private _clock: THREE.Clock;
+  private _delta: number;
```

Comment:
Do we need `_delta` as a class property?

Reply:
I just confirmed and it's not needed as a class property.
This has been added as class property in the first place as when i defined it as const the first time it was always 0 for some reason and when i gave it let it was giving error as it's never reassigned, then i did this. On my way! to define it as animate method property now.

Reply:
Done in [`2923de0`](https://github.com/jupytercad/JupyterCAD/pull/440/commits/2923de004316aa28c93747efddc83d3b2c460546)